tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UserSteps.cs

```diff
@@ -101,6 +101,8 @@ public async Task WhenAUserIsCreated(string organisationName, Table table)
             var payload = JsonConvert.SerializeObject(data);
             using var content = new StringContent(payload, Encoding.UTF8, "application/json");
             _response.Result = await client.PostAsync(new Uri($"{_organisationUrl}/{organisationId}/users"), content);
+
+            _context["emailSent"] = true;
```

Comment:
This is fine but can you remove the unused using from this file, `using FluentAssertions.Equivalency;` while you're here?

Reply:
With pleasure 😄